licenses-binary/LICENSE-BSD-3-clause.txt

```diff
@@ -0,0 +1,27 @@
+
+Copyright (c) 2000-2011 INRIA, France Telecom
```

Comment:
I put one license txt file for each library for BSD-3 because each library has different copyright. Please refer `LICENSE-jcabi.txt, LICENSE-paranamer.txt..etc`. Please create one file for each library.

Reply:
They are all the same license, with the last clause either specifying the actual provider's name, or using a general term. Did you see a different one? I will take a look again.